bcrhp/src/bcrhp/pages/steps/Step7_UploadImage.vue

```diff
@@ -64,10 +65,13 @@ const onFocusOutHandler = function (event: Event) {
 
 const validateField = function (field: HTMLInputElement) {
     console.log(`ID: ${field.id}`);
-    const key: keyof typeof UploadImage = field.id as keyof typeof UploadImage;
+
+    const key: keyof typeof HeritageSite =
```

Comment:
Should this be:
```
const key: keyof typeof UploadImage = field.id as keyof typeof UploadImage;
```

Also - can you please update the "UploadImage" schema to be named `SiteImages`? `Upload` is an action, we should be using descriptive terminology here.